scripts/merge_tract_cat.py

```diff
@@ -126,6 +128,8 @@ def load_tract(repo, tract, patches=None, **kwargs):
         except NoResults as e:
             print(e)
             continue
+        if len(this_patch_merged_cat) == 0:
+            continue
```

Comment:
Maybe add `print('empty patch')` so that it has a similar behavior like `print(e)` above?

Reply:
👍 
I got a bit more elaborate with standardizing the error messages in my final changes.  Thanks for the suggestions.
So now it outputs things like
```
Processing tract 4433, patch 7,4
  No locations for get: datasetType:deepCoadd_ref dataId:DataId(initialdata={'tract': 4433, 'patch': '7,4'}, tag=set())
  No good entries for tract 4433, patch 7,4
Processing tract 4433, patch 7,5
  No locations for get: datasetType:deepCoadd_forced_src dataId:DataId(initialdata={'tract': 4433, 'patch': '7,5', 'filter': 'g'}, tag=set())
  No locations for get: datasetType:deepCoadd_forced_src dataId:DataId(initialdata={'tract': 4433, 'patch': '7,5', 'filter': 'r'}, tag=set())
  No locations for get: datasetType:deepCoadd_forced_src dataId:DataId(initialdata={'tract': 4433, 'patch': '7,5', 'filter': 'i'}, tag=set())
  No locations for get: datasetType:deepCoadd_forced_src dataId:DataId(initialdata={'tract': 4433, 'patch': '7,5', 'filter': 'z'}, tag=set())
  No locations for get: datasetType:deepCoadd_forced_src dataId:DataId(initialdata={'tract': 4433, 'patch': '7,5', 'filter': 'y'}, tag=set())
```